app/views/curation_concerns/multi_volume_works/_members.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if @presenter.scanned_resource_presenters.present? %>
+  <div class="panel panel-default scanned_resources">
+    <div class="panel-heading">
+      <h2>Scanned Resources</h2>
+    </div>
+    <ul>
+      <% @presenter.scanned_resource_presenters.each do |res| %>
+        <li>
+          <%= link_to res.title, main_app.curation_concerns_scanned_resource_path(res.id) %>
```

Comment:
We should probably be using `CurationConcerns::ParentContainer` and just have a ScannedResource partial for this.  I tried that but it wasn't working because the partial expected a counter parameter.  Can we punt that to a new ticket?